Demo/src/main/java/com/paypal/android/api/services/SDKSampleServerAPI.kt

```diff
@@ -26,7 +26,12 @@ private const val WRITE_TIMEOUT_IN_SEC = 30L
 private val DEFAULT_CLIENT_ID: String? = null // = "your-client-id"
 private val DEFAULT_ORDER_ID: String? = null // = "your-order-id"
 
-class SDKSampleServerAPI(baseUrl: String) {
+class SDKSampleServerAPI {
+
+    companion object {
+        // UPCOMING PR: require Merchant enum to be specified via UI layer
```

Comment:
Nitpick so this comment is searchable

```suggestion
        // TODO: - require Merchant enum to be specified via UI layer
```